settings.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" href="style.css">
+	<title>Settings Modal</title>
+</head>
+<body>
+	<div id="modal">
+		<div id="accent-top"></div>
+		<div id="title">Settings</div>
+		<div id="settings">
+			<div class="student-information">
+				Student ID <br/>
+				<input type="text" id="student-id">
+			</div>
+			<div class="student-information">
+				Class ID <br/>
+				<input type="text" id="class-id">
+			</div>
+			<a href="index.html">
+				<div class="button">
```

Comment:
Instead of having a few inputs without a form, lets surround these divs with a form and make this an actual submit button instead (we can do javascript form handling and just mock that it saved the changes)